lib/Screens/Detail.dart.dart

```diff
@@ -0,0 +1,247 @@
+
+class DetailedScreen extends StatefulWidget {
+  final String url;
+  final String text;
+  const DetailedScreen({super.key, required this.url, required this.text});
+
+  @override
+  State<DetailedScreen> createState() => _DetailedScreenState();
+}
+
+class _DetailedScreenState extends State<DetailedScreen> {
+  double height = 50;
+  double width = 50;
+  double height1 = 50;
+  double width1 = 300;
+  Color border_color = Colors.white;
+  Color color = Colors.transparent;
+
+  String? stringResponse;
+  Map? mapResponse;
+  List listResponse = [];
+
+  Future apiCall() async {
+    http.Response response;
+
+    response = await http.get(Uri.parse(widget.url));
+
+    if (response.statusCode == 200) {
+      setState(() {
+        stringResponse = response.body;
+        mapResponse = json.decode(response.body);
+        listResponse = mapResponse!["articles"];
+      });
+    } else {
+      debugPrint("Error in loading!!!");
+    }
+  }
+
+  @override
+  void initState() {
+    apiCall();
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        iconTheme: const IconThemeData(color: Colors.deepPurple),
+        backgroundColor: Colors.white,
+        elevation: 0,
+        leading: const Icon(Icons.menu),
+        actions: const [Icon(Icons.notifications)],
+        title: const Text(
+          "Taza Khabar",
+          style: TextStyle(color: Colors.deepPurple),
+        ),
+      ),
+      body: SingleChildScrollView(
+        child: Column(
+          children: [
+            Row(
+              children: [
+                AnimatedContainer(
+                  duration: const Duration(milliseconds: 20),
+                  height: height1,
+                  width: width1,
+                  child: Padding(
+                    padding: const EdgeInsets.all(8.0),
+                    child: Text(
+                      widget.text,
+                      style: const TextStyle(
+                          fontSize: 30, fontStyle: FontStyle.italic),
+                    ),
+                  ),
+                ),
+                AnimatedContainer(
+                  duration: const Duration(milliseconds: 20),
+                  height: height,
+                  width: width,
+                  decoration: BoxDecoration(
+                      border: Border.all(color: border_color, width: 0.0),
+                      borderRadius: BorderRadius.circular(10),
+                      color: color),
+                  child: Padding(
+                    padding: const EdgeInsets.all(8.0),
+                    child: Row(
+                      children: [
+                        InkWell(
+                          onTap: () {
+                            color = color;
+
+                            Colors.black;
+                            height = 0.0;
+                            width = 0.0;
+                            setState(() {});
```

Comment:
please write the above block in setState, empty setStates are not a good practice